app/src/main/index.js

```diff
@@ -294,7 +296,12 @@ menubar.on('after-create-window', () => {
       // The `move` event is called when the user reselases the mouse button
       // because of that, we need to move the window to it's expected position, since the
       // user will never release the mouse in the *right* position (diff.[x, y] === 0)
-      tray.setHighlightMode('always');
+      if (!wasOpenedAtLogin(app)) {
```

Comment:
This introduces a bug – tray will never be highlighted if this check is in place and the app was opened on system startup 🐛